### PR TITLE
Added 2 Matcall BV devices

### PR DESCRIPTION
**Pull request by sunsear**
Tested with externalConverters on my HassOS installation. Unfortunately I cannot test devices.js in this setup, I have thoroughly checked the syntax, but please check again.

### CODE REVIEW

**Comment by sunsear** [comment]
OK, listing did that syntax check for me ;-)

I have seen of course that my 2 devices could extend from light_onoff_brightness, but I am unable to test that and prove that it works as I need to run them from External Converters in my HassOS setup.

File: devices.js [hunk 1]
```diff
@@ -17524,6 +17524,28 @@ const devices = [
         extend: preset.light_onoff_brightness_colortemp({colorTempRange: [200, 370]}),
     },
 
+    // Matcall BV
+    {
+        zigbeeModel: ['ZG 401224'],
+        model: 'ZG 401224',
+        vendor: 'Matcall bv',
+        description: 'Matcall LED Dimmer driver',
+        exposes: [e.light_brightness()],
```
Comment (Koenkk) [inline devices.js:17533]: For both `exposes` `fromZigbee` and `toZigbee` can be replaced with `extend: preset.light_onoff_brightness(),`

File: devices.js [hunk 2]
```diff
@@ -17524,6 +17524,28 @@ const devices = [
...
+        fromZigbee: [fz.on_off, fz.brightness, fz.ignore_basic_report],
+        toZigbee: [tz.light_onoff_brightness, tz.ignore_transition, tz.ignore_rate, tz.light_brightness_move,
+            tz.light_brightness_step, tz.level_config, tz.power_on_behavior],
+    },
+    {
+        zigbeeModel: ['ZG 430700', 'ZG  430700'],
+        model: 'ZG 430700',
```
Comment (Koenkk) [inline devices.js:17540]: Please remove space for both model numbers (see bottom of page 26: https://www.matcall.nl/wp-content/uploads/documentatie/MC-Easy-systeem.pdf)

Comment (sunsear) [reply devices.js:17540]: These model numbers are the actual reported model numbers in the Zigbee stack. I guess it does make more sense to make them list as the documentation does. On their website they are also referred to without spaces. Will make changes!

File: devices.js (hunk 1 above)
Comment (sunsear) [reply devices.js:17533]: OK will do, although I cannot test that myself

**Comment by sunsear** [comment]
Thanks for review Koen, I have added the requested changes.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**